lib/history_provider.dart

```diff
@@ -7,6 +7,10 @@ import 'package:uuid/uuid.dart';
 
 import 'link_matcher.dart';
 
+part 'providers/browser_history.dart';
+part 'providers/hash_history.dart';
+part 'providers/memory_history.dart';
```

Comment:
had to file in w_router repo instead because apparently we can't file issues on this forked repo :frowning: 
https://github.com/Workiva/w_router/issues/8